src/web/central_command/client/components/AXUI.jsx

```diff
@@ -30,6 +31,9 @@ import OlIcon from 'ol/style/Icon'
 import OlLayerGroup from 'ol/layer/Group';
 import OlSourceOsm from 'ol/source/OSM';
 import OlSourceXYZ from 'ol/source/XYZ';
+import OlTileWMS from 'ol/source/TileWMS';
```

Comment:
I got an error while trying to run the run.sh script because this line is duplicated (line 34, and line 42)